package-lock.json

```diff
@@ -1837,53 +1837,20 @@
       }
     },
     "node_modules/@govuk-pay/pay-js-commons": {
-      "version": "3.8.0",
-      "resolved": "https://registry.npmjs.org/@govuk-pay/pay-js-commons/-/pay-js-commons-3.8.0.tgz",
-      "integrity": "sha512-szo+dxqBJCpM3TGVokB6+YF7sfBrjazq4ISViZyUuwpn/vFz08tG5rXTIZlRfddrACLxoFRdqiWDBPw4sU5sNg==",
+      "version": "3.8.2",
+      "resolved": "https://registry.npmjs.org/@govuk-pay/pay-js-commons/-/pay-js-commons-3.8.2.tgz",
+      "integrity": "sha512-Ezpex31o5tcdQsy7jyBif4HPFUPnNRBUpt8N55/sDcaUA/QQ9wozT2nwYXYF6XwkW4cRrJL6DnrMxy0IGIFFjg==",
       "dependencies": {
         "lodash": "4.17.21",
-        "moment-timezone": "0.5.37",
+        "moment-timezone": "0.5.38",
         "rfc822-validate": "1.0.0",
         "slugify": "1.6.5",
-        "winston": "3.8.1"
+        "winston": "3.8.2"
       },
       "engines": {
         "node": "^16.14.0"
       }
     },
-    "node_modules/@govuk-pay/pay-js-commons/node_modules/readable-stream": {
```

Comment:
Any reason other packages are removed/updated?

Reply:
npm install did that 🤷‍♂️

Reply:
Looks like that is due to the pay-js-commons dependency updates